### PR TITLE
frontend#27 Build project layout.

### PR DESCRIPTION
**Pull request by camman00**
Completed the project layout which has CSS variables at the root. @azpi-arte take a look when you get the chance I put you as the reviewer!

### CODE REVIEW

File: application/web/main.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+:root {
+    --journal-entry: #F3F3F3;
+    --sliding-shelf: #EBEBEB;
+    --main-stroke-color: #A1A1A1;
+    --grey-text-color: #404040;
+    --icon-color: #D7D7D7;
```
Comment (azpi-arte) [inline application/web/main.css:6]: The design didn't end up using the icon-color so its okay to delete this styling.

File: application/web/main.css
Comment (azpi-arte) [inline application/web/main.css:1]: Absolutely Stunning 💪
Amazing Start.

The only thing I would tweak is removing the icon-color CSS.

File: application/web/index.html
Comment (azpi-arte) [inline application/web/index.html:1]: Nice and simple. I like it 👍

**Review by azpi-arte — CHANGES_REQUESTED** [review]
Very great, just needs one quick change

**Comment by camman00** [comment]
Went ahead and merged. Ill create a hot fix for icon styling.

**Pull request merged**